examples/events.py

```diff
@@ -22,17 +22,18 @@ def handler(events):
          "StdinOnce":False,
     }
 
-    container = yield from docker.containers.run(config, name='testing')
+    container = yield from docker.containers.create(config, name='testing')
+    yield from container.start(config)
 
     while True:
         event = yield from queue.get()
-        if event['status'] == 'create':
+        if event['status'] == 'start':
             yield from event['container'].stop()
```

Comment:
This line is still fun.  When `status = create` is the check, this just hangs because it can't stop a non-started container.